package.json

```diff
@@ -1,9 +1,12 @@
 {
-  "name": "FeedHenryHealthMonitoring",
-  "version": "0.1.1",
+  "name": "feedhenryhealthmonitoring",
+  "version": "0.1.2",
   "scripts": {
-    "test": "grunt jshint",
-    "start": "node application.js"
+    "start": "node application.js",
+    "serve": "grunt serve",
+    "unit": "grunt test",
+    "test": "npm run unit",
```

Comment:
why make `test` command to run `npm run init` and then `unit` run `grunt test` and not make `test` call `grunt test` directly?